libs/community/langchain_community/embeddings/huggingface.py

```diff
@@ -329,6 +329,11 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
                 "options": {"wait_for_model": True, "use_cache": True},
             },
         )
+        if response.status_code != 200:
```

Comment:
The exception should not be a generic exception, so one can do retries on 5xx errors, which this code is more likely to raise since it might be involved in repeated API calls

It might make sense to do something like `response.raise_for_json()` so caller code can retrieve the status code and apply re-try logic appropriately